applications/nrf_desktop/src/modules/Kconfig.led_stream

```diff
@@ -8,7 +8,7 @@ menu "LED stream"
 
 config DESKTOP_LED_STREAM_ENABLE
 	bool "Enable LED stream"
-	depends on DESKTOP_LED_ENABLE && DESKTOP_CONFIG_CHANNEL_ENABLE
+	depends on CAF_LED_EVENTS && DESKTOP_CONFIG_CHANNEL_ENABLE
```

Comment:
CONFIG_CAF_LEDS, we actually need the module that controls the HW LEDS

Reply:
Yes, but from SW perspective we need events and nothing more.

Reply:
@pdunaj What is your statement? I remember we talked about that offline.

Reply:
@emob-nordic is right. In the end we don't care who will handle the request. Integrator of nrf_desktop can add proprietary module handling leds.

Reply:
Left with `*_EVENTS`.